packages/external-interfaces/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@smithy/external-interfaces",
+  "version": "1.0.0",
```

Comment:
⚠️ this package must be published immediately upon merge of this PR so downstream generated SDKs can start consuming it.

Reply:
PR could also be split into 

1 - the package to be published
2 - the codegen change